app/scss/partials/_detail.scss

```diff
@@ -73,24 +73,36 @@
 	@include expanding-panel(2px);
 }
 
-.condensed-toggle {
+.toggle-bar:hover {
 	position: absolute;
-	height: 50px;
-	width: 25px;
-	top: 198px;
+	height: 100%;
+	width: 2px;
+	top: 0px;
 	z-index: 2;
 	cursor: pointer;
-	background-position: 7px center;
+	background-color: $ghosty-blue;
+}
+
+.condensed-toggle {
+	position: absolute;
+	height: 24px;
+	width: 13px;
+	border-bottom-right-radius: 15px;
+	border-top-right-radius: 15px;
+	top: 215px;
+	z-index: 3;
+	cursor: pointer;
+	background-position: center;
```

Comment:
Try using this to move the left caret to the left:
```
background-position: calc(50% - 1.75px);
```
you may have to have separate css selector for the right arrow and left.